src/odemis/acq/fastem_conf.py

```diff
@@ -178,3 +178,12 @@ def configure_detector(detector, roc2, roc3):
         logging.warning("Region of calibration doesn't have cell translation parameters.")
     else:
         detector.cellTranslation.value = cell_translation
+
+
+def configure_multibeam(multibeam):
```

Comment:
The arguments of the for each of this configure functions is not the same, so it will be difficult. Otherwise a single configuration yaml file can possibly be used.

Reply:
I'll just leave it like this then